src/test/java/pl/company/foodatu/FoodatuApplicationTests.java

```diff
@@ -1,13 +1,49 @@
 package pl.company.foodatu;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.company.foodatu.plans.PlansClient;
+import pl.company.foodatu.plans.dto.PlanResponse;
+import pl.company.foodatu.plans.utils.PlansTestUtils;
 
-@SpringBootTest
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@RunWith(SpringRunner.class)
 class FoodatuApplicationTests {
 
+	@Autowired
+	PlansClient plansClient;
+
 	@Test
-	void contextLoads() {
+	void contextLoads() { //NOSONAR
+	}
+
+	@Test
+	void addedMeal_then_planContains1MealAndItIsReturnedWithPlan() {
+		//given
+		plansClient.addMealToPlan(PlansTestUtils.SANDWICH_WITH_CHEESE, PlansTestUtils.USER.id(), PlansTestUtils.TODAY);
```

Comment:
Try to use static imports, code will become clearer
```suggestion
		plansClient.addMealToPlan(SANDWICH_WITH_CHEESE, USER.id(), TODAY);
```